docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+  - meson
+    - Version 0.55.3 has been upgraded to version 0.60.3.
+  - mm-common
+    - Version 1.0.1 has been upgraded to version 1.0.3.
+  - mtools
+    - Version 4.0.18 has been upgraded to version 4.0.36.
+  - multipath-tools
+    - Version 0.4.9 has been upgraded to version 0.8.8.
+  - openfst
+    - Version 1.6.5 has been upgraded to version 1.8.1.
+  - paho-mqtt-c
+    - Version 1.3.7 has been upgraded to version 1.3.9.
+  - parted
+    - Version 3.1 has been upgraded to version 3.4.
+  - patch
+    - Version 2.7 has been upgraded to version 2.7.6.
+  - pcre
+    - Version 8.44 has been upgraded to version 8.45.
+  - perl
+    - Version 5.32.0 has been upgraded to version 5.34.0.
+  - pkg-config
+    - Version 0.28 has been upgraded to version 0.29.2.
+  - procps-ng
+    - Version 3.3.16 has been upgraded to version 3.3.17.
+  - psmisc
+    - Version 23.3 has been upgraded to version 23.4.
+  - python3
+    - Version 3.9.1 has been upgraded to version 3.9.10.
+  - re2
+    - Version 20201101 has been upgraded to version 20220201.
+  - re2c
+    - Version 1.1.1 has been upgraded to version 3.0.
+  - rpmlint
+    - Version 1.4 has been upgraded to version 1.11.
+  - rsync
+    - Version 3.1.1 has been upgraded to version 3.2.3.
+  - sgml-skel
+    - Version 0.6 has been upgraded to version 0.7.1.
+  - sqlite
+    - Version 3.33.0 has been upgraded to version 3.36.0.
+  - sudo
+    - Version 1.9.4 has been upgraded to version 1_9_7p2.
+  - tzdata
+    - Version 2020a has been upgraded to version 2021e.
+  - util-linux
+    - Version 2.36 has been upgraded to version 2.37.
+
+
+### Application framework
+
+#### New and changed features
+
+- Tizen boot sequence
+  - A new feature has been ready to control app launch on system boot.
+  - A tool to check the status of the app after booting has been provided.
+- TIDL
+  - The TIDL compiler can generate new API set using the Cion-group API set.
+
+#### Fixes
+
+- App-core
+  - The implementation has been refactored into the C++ language.
+- Pkgmgr-info
+  - A cache for pkginfo-server has been applied to improve the online mode performance of getting package information.
+  - Offline mode performance of getting package information has been improved.
+- App-installer
+  - The preload package installation order in MIC builds has been changed for devices with low storage capacity.
+
+
+### Window and Interaction
+
+#### New and changed features
+
+- Open source
+  - wayland has been upgraded to version 1.20.0.
+  - The libxkb-common has been upgraded to version 1.3.1.
+  - The libinput has been upgraded to version 1.17.0.
+  - The libevdev has been upgraded to version 1.11.0.
+  - The mesa has been upgraded to version 21.2.3.
+- Enlightenment
+  - The Drag and Drop has been added. It provides wl_data_device_manager interface of wayland protocol to the wayland-clients.
+- Vulkan
+  - The vulkan-wsi-layer has been added. It implements Vulkan window system integration extensions.

```

Comment:
It is preferred to not use 'the' before a tool name.
```suggestion
  - Libxkb-common has been upgraded to version 1.3.1.
  - Libinput has been upgraded to version 1.17.0.
  - Libevdev has been upgraded to version 1.11.0.
  - Mesa has been upgraded to version 21.2.3.
- Enlightenment
  - Drag and Drop has been added. It provides wl_data_device_manager interface of wayland protocol to the wayland-clients.
- Vulkan
  - Vulkan-wsi-layer has been added. It implements Vulkan window system integration extensions.

```

Reply:
will accept